index.html

```diff
@@ -0,0 +1,12 @@
+<html>
```

Comment:
super helpful test html files for quick local debugging (this redirects to `test/index.html`)